testing/unclutter-xfixes/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname=unclutter-xfixes
+pkgver=1.2
+pkgrel=0
+pkgdesc="Hides mouse pointer while not in use"
+url="https://github.com/Airblader/unclutter-xfixes"
+arch="all"
+license="MIT"
+depends="libxfixes"
```

Comment:
This is automatically inferred by `abuild` (see the `so:` analysis in the build log) and doesn't need to be specified on its own. :)